### PR TITLE
LUN-203: Rename menu item

### PR DESCRIPTION
**Pull request by Saraharar98**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/components/__tests__/CustomDisciplineMenuItem.spec.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
   mockUseLoadAsyncWithError
 } from '../../testing/mockUseLoadFromEndpoint'
 import wrapWithTheme from '../../testing/wrapWithTheme'
-import CustomDisciplineMenuItem from '../CustomDisciplineMenuItem'
+import CustomDisciplineItem from '../CustomDisciplineItem'
 
 describe('Components', () => {
   describe('CustomDisciplineMenuItem', () => {
```
Comment (SteffiStuffel) [inline src/components/__tests__/CustomDisciplineMenuItem.spec.tsx:15]: Can you please rename here and also the adjust the filename?

File: src/components/__tests__/MenuItem.spec.tsx [hunk 2]
```diff
@@ -24,8 +24,8 @@ describe('Components', () => {
       onPress: () => {}
     }
 
-    const renderMenuItem = (overrideProps: Partial<ComponentProps<typeof MenuItem>> = {}): RenderAPI => {
-      return render(<MenuItem {...defaultMenuItemProps} {...overrideProps} />, {
+    const renderMenuItem = (overrideProps: Partial<ComponentProps<typeof DisciplineItem>> = {}): RenderAPI => {
```
Comment (SteffiStuffel) [inline src/components/__tests__/MenuItem.spec.tsx:27]: Please also rename this function

File: src/components/CustomDisciplineItem.tsx [hunk 3]
```diff
@@ -46,7 +46,7 @@ interface CustomDisciplineMenuItemPropsType {
   refresh: () => void
```
Comment (SteffiStuffel) [inline src/components/CustomDisciplineItem.tsx:46]: This interface (line 41) should also be renamed.

File: src/components/DisciplineItem.tsx [hunk 4]
```diff
@@ -43,7 +43,7 @@ export interface IMenuItemProps {
   onPress: () => void
```
Comment (SteffiStuffel) [inline src/components/DisciplineItem.tsx:43]: Also this interface (line 39) should be renamed. Please name it `DisciplineItemProps`

File: src/components/__tests__/MenuItem.spec.tsx [hunk 5]
```diff
@@ -4,7 +4,7 @@ import { Text } from 'react-native'
 
 import { COLORS } from '../../constants/theme/colors'
 import wrapWithTheme from '../../testing/wrapWithTheme'
-import MenuItem, { IMenuItemProps } from '../MenuItem'
+import DisciplineItem, { IMenuItemProps } from '../DisciplineItem'
 
 describe('Components', () => {
   describe('MenuItem', () => {
```
Comment (SteffiStuffel) [inline src/components/__tests__/MenuItem.spec.tsx:10]: Please rename this, and also in line 11 the variable.

**Review by SteffiStuffel — COMMENTED** [review]
Nice work, you just forget few occurrences. 

Please also rename the PR, it should start with `LUN-203: Ren....`

**Review by klinzo — APPROVED** [review]

File: src/components/__tests__/CustomDisciplineItem.spec.tsx [hunk 6]
```diff
@@ -9,7 +9,7 @@ import {
   mockUseLoadAsyncWithError
 } from '../../testing/mockUseLoadFromEndpoint'
 import wrapWithTheme from '../../testing/wrapWithTheme'
-import CustomDisciplineMenuItem from '../CustomDisciplineMenuItem'
+import CustomDisciplineItem from '../CustomDisciplineItem'
 
 describe('Components', () => {
   describe('CustomDisciplineMenuItem', () => {
```
Comment (SteffiStuffel) [inline src/components/__tests__/CustomDisciplineItem.spec.tsx:15]: please also rename the test here.

File: src/components/__tests__/DisciplineItem.spec.tsx [hunk 7]
```diff
@@ -4,11 +4,11 @@ import { Text } from 'react-native'
 
 import { COLORS } from '../../constants/theme/colors'
 import wrapWithTheme from '../../testing/wrapWithTheme'
-import MenuItem, { IMenuItemProps } from '../MenuItem'
+import DisciplineItem, { DisciplineItemProps } from '../DisciplineItem'
 
 describe('Components', () => {
   describe('MenuItem', () => {
```
Comment (SteffiStuffel) [inline src/components/__tests__/DisciplineItem.spec.tsx:10]: Please rename the test here and also the title-constant in line 17.

**Review by Taggotty — APPROVED** [review]

File: src/components/__tests__/DisciplineItem.spec.tsx [hunk 8]
```diff
@@ -24,43 +24,43 @@ describe('Components', () => {
...
       expect(onPress).not.toHaveBeenCalled()
       const element = getByText('Menu item title')
       fireEvent.press(element)
       expect(onPress).toHaveBeenCalled()
     })
 
     it('should display title passed to it', () => {
-      const { queryByText } = renderMenuItem({})
+      const { queryByText } = renderDisciplineItem({})
       const title = queryByText('Menu item title')
```
Comment (Taggotty) [inline src/components/__tests__/DisciplineItem.spec.tsx:44]:
```suggestion
      const title = queryByText('Discipline Item title')
```

File: src/components/__tests__/DisciplineItem.spec.tsx [hunk 9]
```diff
@@ -24,43 +24,43 @@ describe('Components', () => {
...
         wrapper: wrapWithTheme
       })
     }
 
     it('should call onPress event', () => {
       const onPress = jest.fn()
-      const { getByText } = renderMenuItem({ onPress: onPress })
+      const { getByText } = renderDisciplineItem({ onPress: onPress })
       expect(onPress).not.toHaveBeenCalled()
       const element = getByText('Menu item title')
```
Comment (Taggotty) [inline src/components/__tests__/DisciplineItem.spec.tsx:37]:
```suggestion
      const element = getByText('Discipline Item title')
```

**Review by Taggotty — CHANGES_REQUESTED** [review]

**Review by SteffiStuffel — APPROVED** [review]

**Review by Taggotty — APPROVED** [review]

**Pull request merged**